Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -924,6 +964,38 @@ void MenuBar::UpdateToolsMenu(bool emulation_started)
   }
 }
 
+#if defined(_WIN32)
+#include <Common/FileUtil.h>
+void MenuBar::DumpFrames_FFmpegNotFound_Win32()
+{
+  const QString dol_root_dir = tr(File::GetExeDirectory().c_str());
```

Comment:
Why is `tr` used here? This isn't translateable in any reasonable means. It's a directory path.

Reply:
You probably want `QString::fromStdString` here.

Reply:
This one took me some time to figure out, persistance paid off and after lots of combinations this is what worked. If I remove the tr() it it will not compile: `Error C2248 - 'QString::QString': cannot access private member declared in class 'QString'`
@JosJuice I'll try that.